### DISCUSSION TITLE
Can you require people to resize their browser?

### DISCUSSION THREAD

**Discussion by SaoirseConnorDesai** [jspsych/jsPsych#730] [Q&A] [answered]
Is there a way to require people to resize their browsers before continuing to the experiment?

At the moment it looks like you can completely bypass the resize widget which means some participants aren't able to see the task.

**Comment by jodeleeuw** [comment, marked as answer]
You can use the [exclusion criteria in jsPsych.init()](https://www.jspsych.org/overview/exclude-browser/) to require people to have a certain size browser window before proceeding. The method allows participants to resize their browser to meet the minimum size.

Example:
```js
jsPsych.init({
  ...
  exclusions: {
    min_width: 800,
    min_height: 600
  }
});
```
